docs/source/poutine.rst

```diff
@@ -1,4 +1,4 @@
-Poutines (Pyro Coroutines)
+Poutine (Effect handlers)
```

Comment:
when you remove the old interface are you going to remove poutines entirely and expose them as pyro primitives ie `with pyro.scale(...)` or keep the existing namespace? 

>Neat, PR/issue number 1000!

"24 more until we reach a nice round number" :P

Reply:
I'm going to keep the `poutine` namespace, but the behavior of the aliases (`trace`, `replay`, etc.) will now be like the behavior of their respective `Messenger`s as of this PR.